quotephase.go

```diff
@@ -0,0 +1,137 @@
+//
+//
+// File generated from our OpenAPI spec
+//
+//
+
+package stripe
+
+// If set to `new`, the billing_cycle_anchor of the subscription is set to the start of the phase when entering the phase. If unset, then the billing cycle anchor is automatically modified as needed when entering the phase. For more information, see the billing cycle [documentation](https://stripe.com/docs/billing/subscriptions/billing-cycle).
+type QuotePhaseBillingCycleAnchor string
+
+// List of values that QuotePhaseBillingCycleAnchor can take
+const (
+	QuotePhaseBillingCycleAnchorReset QuotePhaseBillingCycleAnchor = "reset"
```

Comment:
Already flagged in other code review but flagging to not forget.

Reply:
Going to follow up in a second PR. Great catch!